### DISCUSSION TITLE
Automatic IOTA entries

### DISCUSSION THREAD

**Discussion by soakland5** [magicbug/Cloudlog#3194] [Q&A]
I see that many of my QSOs automatically enter what it thinks is the IOTA # of the callsign entered. Is there a way to turn this off? Many of them are incorrect. Nice feature, but just wanted to know if it can be turned off.

Thanks
Scott N1IA

**Comment by magicbug** [comment]
Believe this data is coming from QRZ but I'll double check later

**Reply by soakland5** [reply]
Any luck finding out this info? Thanks, Scott
